httpcore/_backends/curio.py

```diff
@@ -0,0 +1,202 @@
+import select
+from ssl import SSLContext, SSLSocket
+from typing import Optional
+
+import curio
+import curio.io
+
+from .._exceptions import (
+    ConnectError,
+    ConnectTimeout,
+    ReadError,
+    ReadTimeout,
+    WriteError,
+    WriteTimeout,
+    map_exceptions,
+)
+from .._types import TimeoutDict
+from .._utils import get_logger
+from .base import AsyncBackend, AsyncLock, AsyncSemaphore, AsyncSocketStream
+
+logger = get_logger("curio_backend")
```

Comment:
The standard practice for loggers seems to be just using `__name__` (which here will be something like `httpcore._backends.curio`). This way, any logging setup on a parent module (here `httpcore`) is used automatically.

So this should be:

```suggestion
logger = get_logger(__name__)
```